### PR TITLE
Updated validation on checkboxes

### PR DESCRIPTION
**Pull request by martyncolmer**
Added custom validation to deal with exclusive checkboxes.

### CODE REVIEW

File: eq-author-api/src/validation/customKeywords/validateExclusiveCheckbox.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+const {
+  getOptionById,
+  getExpressionGroupByExpressionId,
+} = require("../../../schema/resolvers/utils");
+const { filter } = require("lodash");
+
+module.exports = function(ajv) {
+  ajv.addKeyword("validateExclusiveCheckbox", {
+    validate: function isValid(
+      otherFields,
```
Comment (tjamesmac) [inline eq-author-api/src/validation/customKeywords/validateExclusiveCheckbox.js:15]: Think you can delete this as it's not used

File: eq-author-api/src/validation/customKeywords/validateExclusiveCheckbox.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      entityData,
+      fieldValue,
+      dataPath,
+      parentData,
+      fieldName,
```
Comment (tjamesmac) [inline eq-author-api/src/validation/customKeywords/validateExclusiveCheckbox.js:20]: And this one

Comment (tjamesmac) [reply eq-author-api/src/validation/customKeywords/validateExclusiveCheckbox.js:20]: Well that's not helpful. I think `otherFields` and `fieldName` are the ones that aren't used

File: eq-author-api/src/validation/customKeywords/validateExclusiveCheckbox.js (hunk 1 above)
Comment (martyncolmer) [reply eq-author-api/src/validation/customKeywords/validateExclusiveCheckbox.js:15]: I did originally, but the arguments are positional.

Comment (tjamesmac) [reply eq-author-api/src/validation/customKeywords/validateExclusiveCheckbox.js:15]: ah right

**Pull request merged**